.github/workflows/ci.yml

```diff
@@ -87,9 +87,7 @@ jobs:
             node: 20
     runs-on: ${{ matrix.os }}
     steps:
-      # Workaround for: https://github.com/bazel-contrib/bazel-lib/issues/968.
-      # TODO(devversion): Remove when Aspect lib issue is fixed.
-      - run: choco install gzip
+      - uses: Vampire/setup-wsl@v4
```

Comment:
This should be locked to a specific commit

Reply:
Agreed. Will move this out anyway to a shared action; but Renovate should take care